umbra-js/src/classes/Umbra.ts

```diff
@@ -734,7 +734,7 @@ async function* recursiveGraphFetch(
   overrides?: GraphFilterOverride
 ): AsyncGenerator<any[]> {
   // retrieve the last ID we collected to use as the starting point for this query
-  const fromTimestamp = before.length ? (before[before.length - 1].timestamp as string | number) : false;
```

Comment:
Fetching is ending up in some kind of a loop.

https://github.com/ScopeLift/umbra-protocol/assets/61768337/926aca5f-177d-4844-a2be-36a4f8513711

Reply:
https://github.com/ScopeLift/umbra-protocol/pull/594/commits/35983bb1b34f5f3dcaec573dd39a19a51fcd046e

Reply:
https://github.com/ScopeLift/umbra-protocol/assets/61768337/4998c478-ad31-42d0-a2e4-43ae27a6f0fe

It's now not showing older txs that, on production, shows up after the second/full scan

Reply:
> 

Couldn't reproduce this more than once. Can ignore it for now.